examples/webhooks/webhook_server.go

```diff
@@ -2,9 +2,10 @@ package main
 
 import (
 	"fmt"
-	"log"
 	"net/http"
 
+	log "github.com/sirupsen/logrus"
```

Comment:
We're changing the logger?

Reply:
The CLi within this repo already uses logrus, and so I'd say this is less of a change and probably more of a unification.